bunsen/bunsen-core/src/main/java/com/cerner/bunsen/FhirContexts.java

```diff
@@ -72,4 +154,28 @@ public static FhirContext forR4() {
 
     return contextFor(R4);
   }
+
+  /**
+   * Similar to {@link #forR4()}}, the {@code structureDefinitionsPath} is the list of paths
+   * containing the structure definition files to be loaded into the FHIRContext.
+   *
+   * @param structureDefinitionsPath the list of paths containing the structure definition files
+   * @return a builder for encoders.
+   */
+  public static FhirContext forR4(List<String> structureDefinitionsPath) {
+
+    return contextFor(R4, structureDefinitionsPath);
+  }
+
+  /**
+   * Removes all the FhirContext and the corresponding mappings for the given fhirVersionEnum.
+   *
+   * @param fhirVersionEnum the fhir version for which the details are to be removed
+   */
+  public static void deRegisterFhirContext(FhirVersionEnum fhirVersionEnum) {
```

Comment:
Yeah, I think it makes sense. These were needed for testing, otherwise during `mvn clean install` the test cases failing because the profiles loaded as part of the previous test classes was being reused in the current test class and was throwing conflict. I have annotated the method with `@VisibleForTesting` and left it for now. Do you have any other suggestions?